bin/.travis/enable_swap.sh

```diff
@@ -0,0 +1,3 @@
+sudo /bin/dd if=/dev/zero of=/var/swap.1 bs=1M count=4096
```

Comment:
I prefer to have explicit interpreter in all scripts and also some verbose information:

```suggestion
#!/bin/bash

echo '> Enabling swap...'
sudo /bin/dd if=/dev/zero of=/var/swap.1 bs=1M count=4096
```